gapis/api/vulkan/templates/vk_spy_helpers.cpp.tmpl

```diff
@@ -349,6 +353,56 @@ uint32_t VulkanSpy::SpyOverride_vkCreateDevice(VkPhysicalDevice physicalDevice,
             {{end}}
         {{end}}
     }
+
+    #if COHERENT_TRACKING_ENABLED
+        if (!mMemoryTracker.IsInstalled()) {
+            mMemoryTracker.EnableMemoryTracker();
+
+            VkInstance instance = PhysicalDevices[physicalDevice]->mInstance;
+            auto& instance_functions = mImports.mVkInstanceFunctions[instance];
+            VkPhysicalDeviceMemoryProperties props;
+            instance_functions.vkGetPhysicalDeviceMemoryProperties(physicalDevice, &props);
+            uint32_t coherent_bit = static_cast<uint32_t>(-1);
+            for (uint32_t i = 0 ; i < props.mmemoryTypeCount; ++i) {
+                if ((props.mmemoryTypes[i].mpropertyFlags & (
+                    VkMemoryPropertyFlagBits::VK_MEMORY_PROPERTY_HOST_COHERENT_BIT |
+                    VkMemoryPropertyFlagBits::VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT)) ==
+                            (VkMemoryPropertyFlagBits::VK_MEMORY_PROPERTY_HOST_COHERENT_BIT |
+                             VkMemoryPropertyFlagBits::VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT)) {
+                    coherent_bit = i;
+                }
+            }
+            if (coherent_bit == static_cast<uint32_t>(-1)) {
+                return VkResult::VK_ERROR_INITIALIZATION_FAILED;
+            }
+            uint32_t pagesize = 4096;
```

Comment:
nit: Magic number alert. There's at lease one other declaration of this in `memory_manager.cc`.
Would be nice if there was one place to hold this.

Reply:
I think I can actually fix this as well by exposing the "PageSize()" method from the memory tracker